libs/base/Syntax/PreorderReasoning.idr

```diff
@@ -35,3 +35,13 @@ example x =
    ~~ x+1  ...( plusZeroRightNeutral $ x + 1 )
    ~~ 1+x  ...( plusCommutative x 1          )
 -}
+
+-- Smart constructors
+public export
+(..<) : (y : a) -> {x : a} ->
+    y = x -> Step x y
```

Comment:
Is there a reason this is not using heterogeneous equality like the following combinator?

Reply:
Yes, `Builtins.sym` is homogeneous.
I've changed that now, but we need to use a local copy until the next release to enable bootstrapping